doc: Update documentation for GNSS assistance (#12459)
nrfconnect/sdk-nrf: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst

When GNSS has not been in use for a while or if the device is in relatively weak signaling conditions, it might take longer to acquire a fix.
To improve this, Nordic Semiconductor has implemented the following methods for acquiring a fix in a shorter time:

•	A-GNSS or P-GPS or a combination of both
•	Low accuracy mode

Assisted GNSS (A-GNSS)
----------------------

A-GNSS is commonly used to improve the Time to first fix (TTFF) by utilizing a connection (for example, over cellular) to the internet to retrieve the Almanac and Ephemeris data.

> A connection to an internet server that has the Almanac and Ephemeris data is several times quicker than using the slow 50 bps data link to the GNSS satellites.

Author: No, no need to capitalize those.

P-GPS enables devices to determine the exact orbital location of the satellite without connecting to the network every two hours with a trade-off of reduced accuracy of the calculated position over time.
Note that P-GPS requires more memory compared to regular A-GNSS.

Also, note that due to satellite clock inaccuracies, not all functional satellites will have ephemerides data valid for two weeks in the downloaded P-GPS package.
This means that the number of satellites having valid predicted ephemerides reduces in number roughly after ten days.
Hence, the GNSS module needs to download the ephemeris data from the satellite broadcast if no predicted ephemeris is found for that satellite to be able to use the satellite.

.. note::
   |gps_tradeoffs|

nRF Cloud compared with SUPL library
------------------------------------

* The :ref:`lib_nrf_cloud_agps` library is more efficient to use when compared to the :ref:`SUPL <supl_client>` library, and the latter takes a bit more memory on the device.
* With nRF Cloud, the data is encrypted, whereas SUPL uses plain socket.

> * nRF Cloud also support assistance for QZSS satellites, while SUPL is limited to GPS.

Reviewer: ```suggestion
* nRF Cloud also supports assistance for QZSS satellites, while SUPL is limited to GPS.
```

.. note::
   |gps_tradeoffs|

nRF Cloud compared with SUPL library
------------------------------------

* The :ref:`lib_nrf_cloud_agps` library is more efficient to use when compared to the :ref:`SUPL <supl_client>` library, and the latter takes a bit more memory on the device.

> * With nRF Cloud, the data is encrypted whereas SUPL uses plain socket.

Reviewer: ```suggestion
* With nRF Cloud, the data is encrypted, whereas SUPL uses plain socket.
```

> * nRF Cloud supports assistance also for QZSS satellites, SUPL only for GPS.

Reviewer: ```suggestion
* nRF Cloud also supports assistance for QZSS satellites. SUPL applies only to GPS.
```

Author: As discussed, I'll make it "nRF Cloud also support assistance for QZSS satellites, while SUPL is limited to GPS".

There are many examples in |NCS| that use GNSS.
Following is a list of the samples and applications with some information about the GNSS usage:

> * The :ref:`asset_tracker_v2` application uses nRF Cloud for A-GNSS or P-GPS or a combination of both.

Reviewer: ```suggestion
* The :ref:`asset_tracker_v2` application uses nRF Cloud for A-GNSS, P-GPS, or a combination of both.
```

  The application obtains GNSS fixes and transmits them to nRF Cloud along with sensor data.

> * The :ref:`serial_lte_modem` application uses AT commands to start and stop GNSS and has support for nRF Cloud A-GNSS and P-GPS.

Reviewer 2: ```suggestion
* The :ref:`serial_lte_modem` application uses AT commands to start and stop GNSS and supports nRF Cloud A-GNSS and P-GPS.
```

  The application displays tracking and GNSS fix information in the serial console.

> * The :ref:`gnss_sample` sample does not use assistance by default but can be configured to use nRF Cloud A-GNSS or P-GPS or a combination of both.

Reviewer: ```suggestion
* The :ref:`gnss_sample` sample does not use assistance by default but can be configured to use nRF Cloud A-GNSS, P-GPS, or a combination of both.
```

There are many examples in |NCS| that use GNSS.
Following is a list of the samples and applications with some information about the GNSS usage:

> * :ref:`asset_tracker_v2` application - Uses nRF Cloud for A-GNSS or P-GPS or a combination of both. The application obtains GNSS fixes and transmits them to nRF Cloud along with sensor data.
> * :ref:`serial_lte_modem` application - Uses AT commands to start and stop GNSS and has support for nRF Cloud A-GNSS and P-GPS. The application displays tracking and GNSS fix information in the serial console.
> * :ref:`gnss_sample` sample - Does not use assistance by default but can be configured to use nRF Cloud A-GNSS or P-GPS or a combination of both. The sample displays tracking and fix information as well as NMEA strings in the serial console.

Reviewer 2: ```suggestion
* The :ref:`asset_tracker_v2` application uses nRF Cloud for A-GNSS or P-GPS or a combination of both.
  The application obtains GNSS fixes and transmits them to nRF Cloud along with sensor data.

* The :ref:`serial_lte_modem` application uses AT commands to start and stop GNSS and has support for nRF Cloud A-GNSS and P-GPS.
  The application displays tracking and GNSS fix information in the serial console.

* The :ref:`gnss_sample` sample does not use assistance by default but can be configured to use nRF Cloud A-GNSS or P-GPS or a combination of both.
  The sample displays tracking and fix information as well as NMEA strings in the serial console.
```
